dds_web/utils.py

```diff
@@ -58,3 +70,184 @@ def format_byte_size(size):
         else:
             break
     return f"{size:.2} {suffix}"
+
+
+def invoice_units():
+    """Get invoicing specification from Safespring, calculate and save GBHours and cost for each
+    facility and project."""
+
+    app.logger.debug("Calculating invoicing info...")
+
+    # Create invoice specification
+    # TODO (ina): Change to Safespring API call
+    parent_dir = pathlib.Path("").parent
+
+    # From safespring
+    old_file = parent_dir / pathlib.Path("development/invoicing/safespring_invoicespec.csv")
+
+    current_time = timestamp(ts_format="%Y-%m-%d_%H-%M-%S")
+    to_file = old_file
+    # to_file = parent_dir / pathlib.Path(f"development/invoicing/{current_time}.csv") # TODO (ina): uncomment later
+
+    # shutil.copy(old_file, to_file)    # TODO (ina): uncomment later
+
+    # Get data
+    csv_contents = pandas.read_csv(to_file, sep=";", header=1)
```

Comment:
If `pandas` is only used to read the csv file, its better to look into [csv](https://docs.python.org/3/library/csv.html) package. It light weight compared to pandas

Reply:
It's also to get specific rows, easier with `pandas`. But I will look into changing to `csv` instead.